spec/factories/pages.rb

```diff
@@ -0,0 +1,8 @@
+FactoryGirl.define do
+
+  factory :page do
+    sequence(:page_name) { |n| "page_name_#{n}" }
+    title { |page| page.page_name.to_s.titleize }
+    body 'body'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.